### ISSUE TITLE
[med] JAWS+Chrome: Chat: New chat menu button has a non-descriptive label

### ISSUE THREAD

**Issue by ogumen** [Expensify/App#5657]
## Action Performed:
Prerequisite: user has created a personal account and is logged into same.

1. Using JAWS + Chrome, open https://staging.new.expensify.com
2. Tab over to the New Chat floating action button ( + ) in the chat history panel on the left side and note how it is announced.

## Expected Result:
The element functions as a menu button. It should have a clear name that describes its purpose so users understand what input is required when it is announced with it's programmatic role and state, e.g. "New chat or money request. Menu button [collapsed/expanded]."

## Actual Result:
The button label is not descriptive, the element is announced as 'New Chat (floating action), button' with an incorrect button type and no state announced (issue #5187601).

HTML:
`<div aria-label="New Chat(Floating Action)" role="button" tabindex="0" class="css-1dbjc4n r-1awozwy r-1loqt21 r-1777fci r-u8s1d r-1otgn73" style="background-color: rgb(3, 212, 124); border-radius: 999px; bottom: 34px; height: 52px; right: 20px; transform: rotate(0deg); width: 52px;"><svg xmlns="http://www.w3.org/2000/svg"
viewBox="0 0 20 20" width="20" height="20" fill="rgba(255, 255, 255, 1)"><path d="M16.9 8.8h-5.6V3.1c0-.4-.2-.6-.6-.6H9.4c-.4 0-.6.3-.6.6v5.6H3.1c-.3.1-.6.3-.6.7v1.2c0 .4.2.6.6.6h5.6v5.6c0 .4.2.6.6.6h1.2c.4 0 .6-.2.6-.6v-5.6h5.6c.4 0 .6-.2.6-.6V9.4c.2-.4-.1-.6-.4-.6z"></path></svg></div>`

## Workaround:
No workaround

## Area issue was found in:
Chat page

## Failed WCAG checkpoints
2.4.6

## User impact:
The screen reader users will not get an accurate indication of what functions are available in the menu when it is expanded.

## Suggested resolution:
Use a descriptive label that more accurately describes the actions that can be performed in the menu. e.g. 'New chat or money request'. Avoid using industry (design) terminology or using the accessible name for internal notation, i.e. '(Floating Action)'. Most users are not accustomed to its usage.
https://www.w3.org/WAI/WCAG21/Techniques/general/G131.html
Note: Issues related to the programmatic button type and state announcements is reported under #5187601

## Platform:
Where is this issue occurring?

- Web

**Version Number:**
**Reproducible in staging?:**Yes
**Reproducible in production?:**
**Notes/Photos/Videos:** Any additional supporting documentation
**Issue reported by:**

https://user-images.githubusercontent.com/88733897/136062352-ba77cf98-31dc-4ece-b2fb-a2d09bebade7.mp4

![Bug5186645_Chrome_Jaws-Chat-New_chat_menu_button_has_non-descriptive_name](https://user-images.githubusercontent.com/88733897/136062362-2af0af1f-edd8-433e-bfd7-e0133bee97a9.png)

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)
